core/src/modules/optionHelper.ts

```diff
@@ -16,7 +16,7 @@ import { LockableHelper } from "./lockableHelper";
 import { plural } from "pluralize";
 import { modelName } from "./modelName";
 
-export const ObfuscatedPasswordString = "__ObfuscatedPassword";
+export const ObfuscatedOptionString = "__ObfuscatedOption";
```

Comment:
This change is only between the local API and UI - shouldn't cause any breaking changes